packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -0,0 +1,89 @@
+@import './variables';
+
+:root {
+  --terra-consumer-body-background: url('https://wallpapercave.com/wp/5ynt7Gb.jpg');
+  --terra-consumer-nav-active-item-background: var(--terra-consumer-body-background);
+  --terra-consumer-nav-item-color: #fff;
+
+  --terra-consumer-nav-width: 300px;
```

Comment:
Should the nav(300px) and nav container(320 in _variabless.scss ) be consistent ?